src/ng/components/toolbar/toolbar.top.html

```diff
@@ -1,14 +1,27 @@
-<div class="toolbar toolbar-top">
-	<q-grid:sortbar></q-grid:sortbar>
-	<q-grid:groupbar></q-grid:groupbar>
-	<q-grid:pivotbar></q-grid:pivotbar>
-	<q-grid:progress></q-grid:progress>
-	<!--<q-grid:filter>-->
-		<!--&lt;!&ndash; filter placeholder &ndash;&gt;-->
-		<!--<div class="filter"></div>-->
-	<!--</q-grid:filter>-->
-	<q-grid:options>
-		<!-- options placeholder -->
-		<div class="options">&#8942;</div>
-	</q-grid:options>
+<div class="toolbar toolbar-top" layout="column">
+	<div layout="row">
+		<div flex="100"
```

Comment:
These attributes are ng-material specific. Probably we should keep it in theme templates.